src/django_iseries/operations.py

```diff
@@ -69,6 +69,11 @@ def check_aggregate_support(self, aggregate):
         elif aggregate.sql_function == 'VAR_SAMP':
             raise NotImplementedError("sample variance function not supported")
 
+    def convert_booleanfield_value(self, value, expression, connection):
+        if value in (0, 1):
```

Comment:
2 things:

1. you declare the `expression` and `connection` parameters, but these aren't used, are they automatically passed when appending to the `converters`? If not: can't they just be omitted?
2. the method is called `convert_booleanfield_value`, but in case it's not a `0` or `1`, you just return the value as is, without actually converting. Should this not raise an exception then? Or should the method not be named differently?

Reply:
This is an existing interface from Django itself that I implement... so I don't have much of a choice here :). And the whole bug was triggered because 1 and 0 were passed as is. Because str(0) is something totally different than str(False), for example.

Reply:
So is it then okay to just return the value as is, just in case it isn't a `0` or `1` (even though this shouldn't happen)?